### DISCUSSION TITLE
Sponsoring now available!

### DISCUSSION THREAD

**Discussion by georglauterbach** [docker-mailserver/docker-mailserver#2522] [Announcements]
**Sponsoring is now available for DMS!** If you're using DMS in production, please consider financial support. Have a look at the [repository page of DMS](https://github.com/docker-mailserver/docker-mailserver) and check out the "Sponsors" button at the top.

---

To be as transparent in this discussion as possible, I'd like to point out some details everyone is entitled to know when it comes to sponsoring DMS: There was a private maintainers discussion about sponsoring that only maintainers can read. When I opened the thread and decided to make it private, I did this to allow every maintainer to express their opinion in the team, and in the team only. Money can be a very delicate topic. **The project is not going to die or going to be unmaintained in the near future!** It is my personal belief though that this is a nice way of saying thank you. **All maintainers put great efforts into this project!** I just happen to be the only one (currently) to put my profile in the sponsors list. **Please feel free to express concerns in this discussion** if you have any!

**Comment by ontheair81** [comment]
Sponsoring seems to be meant for companies and organizations, using DMS in production. Is there a way for individuals, using DMS just for family and friends, to give a donation?
I was thinking about this already, and although I can't give much, I would still like to make a small donation as I really want to keep this project alive.

**Reply by georglauterbach** [reply]
Hey @ontheair81, thank you for considering a donation :) Sponsoring via GitHub is not limited to companies or organizations - individuals can just as easy make a donation:  <https://github.com/sponsors/georglauterbach>. You can make a donation via PayPal or with a credit card. Every donation counts, especially small ones :D Thank you very much in advance!

**Reply by georglauterbach** [reply]
@ontheair81 there we're some concerns raised in the maintainers team that my wording led you to the conclusion the project was dying or may be unmaintained in the near future. This is not the case, and I am very sorry if I led you to believe that. I will refund your donation if you want to, and I just want to be transparent here. Your donation has helped me in particular, and I would like to express my gratitude again :)

**Reply by ontheair81** [reply]
All fine ;-) From time to time I donate to several FOSS projects which I like to support; DMS is one of them.
As I said before, I just can do smaller donations, but in my opinion it is just fair to contribute my part for great software and the people maintaining it - and I encourage other users to do the same.
